cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/chart_integration_test.go

```diff
@@ -681,8 +767,21 @@ func TestKindClusterAvailablePackageEndpointsForOCI(t *testing.T) {
 				t.Fatal(err)
 			}
 			// wait until this repo reaches 'Ready'
-			if err = kubeWaitUntilHelmRepositoryIsReady(t, repoName); err != nil {
-				t.Fatal(err)
+			err = kubeWaitUntilHelmRepositoryIsReady(t, repoName)
+			if !tc.unauthorized {
+				if err != nil {
+					t.Fatal(err)
+				}
```

Comment:
I'm seeing this pattern repeated a lot through our code base, have you considered using `stretchr/testify` library, which is already included as a project dependency, and would make tests (IMHO) easier to write, read and maintain? In this case, we'd replace the three lines with just `require.NotNil(err)`.

Reply:
I haven't considered it due to I did not know it existed. I don't have any objection to using it